resolwe_bio/processes/fusion_calling/arriba.py

```diff
@@ -83,6 +83,13 @@ class Input:
             required=False,
         )
 
+        known_functions_file = DataField(
```

Comment:
Shoud be `known_fusions_file`. Rename the input also below.

Reply:
Ah, autocomplete (aka copilot) at its best 🙈